### PR TITLE
Course service retrieve all

### PR DESCRIPTION
**Pull request by tobidub**
Course service retrieve all added on top PR #51

### CODE REVIEW

**Comment by AgentEnder** [comment]
@tobidub this is missing the unit tests for your service method

File: OtripleS.Web.Api/Services/Courses/CourseService.cs [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+            ValidateStorageCourse(maybeCourse, courseId);
+
+            return await this.storageBroker.DeleteCourseAsync(maybeCourse);
+        });
+
+        public IQueryable<Course> RetrieveAllCourses() =>
+        TryCatch(() =>
+        {
+            IQueryable<Course> storageCourses = this.storageBroker.SelectAllCourses();
+			ValidateStorageCourses(storageCourses);
```
Comment (hassanhabib) [inline OtripleS.Web.Api/Services/Courses/CourseService.cs:48]: Fix your code indentation here please

Comment (tobidub) [reply OtripleS.Web.Api/Services/Courses/CourseService.cs:48]: Looks like this is a Tabs/Spaces thing... 
In my IDE the Code looks fine. I tried to used "Format Document". Did it help?

**Comment by hassanhabib** [comment]
@tobidub  there's no logic tests in this PR.

**Pull request merged**